kedro-datasets/setup.py

```diff
@@ -59,6 +59,10 @@ def _collect_requirements(requires):
     [
         POLARS, "pyarrow>=4.0", "xlsx2csv>=0.8.0", "deltalake >= 0.6.2"
     ],
+    "polars.LazyPolarsDataset":
```

Comment:
Should `EagerPolarsDataset` be added here as well? If someone uses that directly I'm guessing the requirements would now not be picked up?